istio-telemetry/tracing/templates/service.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: List
```

Comment:
I know this is in master, but this is weird? Why do we have List?

Reply:
Actually I am not sure about why the second Service named tracing in the list exist? might need some confirmation from the owner.

Reply:
@howardjohn Pending merge: https://github.com/istio/istio/pull/12669

@richardwxn The `tracing` service provides a provider independent service that can be used to access the tracing UI - e.g. as referenced in https://istio.io/docs/tasks/telemetry/gateways/

Reply:
Yeah I think we should copy the changes from 12669 here, no reason to have this as a List. Thanks @objectiser

Reply:
Removed the List part